compiler/rustc_parse/src/parser/expr.rs

```diff
@@ -3771,7 +3771,7 @@ impl<'a> Parser<'a> {
                     id: DUMMY_NODE_ID,
                     is_placeholder: false,
                 },
-                TrailingToken::MaybeComma,
+                false,
```

Comment:
I'd expect `next_token == token::Comma` here because fields in `Struct { field1, ..., fieldN }` are comma-separated.

Reply:
Oh, that one should be `this.token == token::Comma`.

Reply:
I have fixed this one now.